docs/guides/deploy-guide/manager.md

```diff
@@ -167,41 +180,42 @@ This is recommended.
    ./run.sh reboot
    ```
 
-### Step 4: Deploy the manager service
+### Step 5: Deploy the manager service
 
-1. Transfer the configuration repository.
+1. Push your local configuration repo state of your seed node to to the git server
+2. Transfer the configuration repository.
 
    ```
    ./run.sh configuration
    ```
 
-2. Deploy the Traefik service. This is optional and only necessary if the Traefik service is to be used.
+3. Deploy the Traefik service. This is optional and only necessary if the Traefik service is to be used.
 
    ```
    ./run.sh traefik
    ```
 
-3. Deploy the Netbox service. This is optional and only necessary if the Netbox service is to be used.
+4. Deploy the Netbox service. This is optional and only necessary if the Netbox service is to be used.
 
    ```
    ./run.sh netbox
    ```
 
-4. Deploy the manager service.
+5. Deploy the manager service.
 
    ```
    ./run.sh manager
    ```
 
-### Step 5: Set vault password on the manager service
+### Step 6: Set vault password on the manager service
 
 Finally, the Ansible Vault password is made known on the manager node. Before that, log in to the manager node
 with the `dragon` user.
 
 ```
+ssh -o IdentitiesOnly=yes -i  ../../secrets/id_rsa.operator dragon@YOUR_MANAGER_NODE
```

Comment:
Use id_rsa.operator instead of ../../secrets/id_rsa.operator. The private key is available at this point.

Reply:
Same as above.